admin/app/views/workarea/admin/product_rules/fields/_category.html.haml

```diff
@@ -2,4 +2,5 @@
 Name
 = select_tag 'product_rule[operator]', options_for_select([['equal', 'equal'], ['not equal', 'not_equal']], rule.operator)
 - categories = Workarea::Catalog::Category.any_in(id: rule.terms)
-= select_tag 'product_rule[value]', options_from_collection_for_select(categories, 'id', 'name', rule.terms), multiple: true, data: { remote_select: { source: catalog_categories_path, options: { placeholder: 'Choose a category...' } }.to_json }
+- current = Workarea::Search::Admin.for(rule.product_list)
```

Comment:
if you did the part to get the ES document ID in the controller, you could write a test for it!